pkg/validators/validator.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validators
+
+import "github.com/k8stopologyawareschedwg/deployer/pkg/validator"
+
+type Validator interface {
```

Comment:
do we need this abstraction already?

Reply:
https://github.com/openshift-kni/numaresources-operator/pull/152#discussion_r810852170

Reply:
ok, so IIUC we are trying to be future-proof here. I'd rather go lightweight.

Reply:
ok. I will change it.